src/utils/parser/tests/__snapshots__/getSymbols.js.snap

```diff
@@ -56,6 +56,30 @@ variables
 [(28, 12), (28, 18)]   person "
 `;
 
+exports[`Parser.getSymbols call sites 1`] = `
+"properties
+
+member expressions
+[(2, 8), (2, 9)] [(2, 0), (2, 9)]  c 
+[(2, 4), (2, 5)] [(2, 0), (2, 5)]  b 
+call expressions
+   undefined 
```

Comment:
it looks like we might be getting the member expressions and not the call expressions with this change?